components/style/values/generics/grid.rs

```diff
@@ -556,3 +557,93 @@ impl<T: ToCss> ToCss for TrackList<T> {
         Ok(())
     }
 }
+
+/// The `<line-name-list>` for subgrids.
+///
+/// `[ <line-names> | repeat(<positive-integer> | auto-fill, <line-names>+) ]+`
+/// **NOTE; This is deprecated!**
```

Comment:
How is this deprecated? Didn't it just got moved to a different greater version of the spec?